### PR TITLE
Never merge your own PRs

### PR DESCRIPTION
**Pull request by Loco1982**
Added warning about self merging PRs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,6 +165,7 @@ As you already installed GNU-tools, there's no need to install Linux on a VM (Vi
 ### Git
 
 *Estimate reading time: 7-8 hours*
+**No one** should merge his/her own PR. The merge should **always** be done by another person (co-worker or client).
```
Comment (facundoolano) [inline README.md:168]: not sure if you followed up on the channel discussion, but I think it would be good to convey that the important thing is not who clicks the merge button but that no PR is merged without review (what the actual review flow is may change on a per project basis, and it may be ok to merge your own PR as long as it has been approved by someone else)

Comment (unbalancedparentheses) [reply README.md:168]: @Loco1982 please update the text based on @facundoolano's comments. thanks

Comment (Loco1982) [reply README.md]: I updated the text taking into account @facundoolano comment.

**Review by facundoolano — APPROVED** [review]

**Review by juanbono — APPROVED** [review]
